examples/text_to_image/train_text_to_image.py

```diff
@@ -278,24 +278,19 @@ def __init__(self, parameters: Iterable[torch.nn.Parameter], decay=0.9999):
         self.decay = decay
         self.optimization_step = 0
 
-    def get_decay(self, optimization_step):
-        """
-        Compute the decay factor for the exponential moving average.
-        """
-        value = (1 + optimization_step) / (10 + optimization_step)
-        return 1 - min(self.decay, value)
-
     @torch.no_grad()
     def step(self, parameters):
         parameters = list(parameters)
 
         self.optimization_step += 1
-        self.decay = self.get_decay(self.optimization_step)
+
+        # Compute the decay factor for the exponential moving average.
+        current_decay = (1 + self.optimization_step) / (10 + self.optimization_step)
```

Comment:
(nit) this is not the current decay, current decay is 

```python
min(
                decay,
                (1 + self.optimization_step) / (10 + self.optimization_step)
            )
```

Reply:
You are right! I didn't like `value` because it doesn't mean anything, but will go back to it because can't think of anything better right now.